<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>David Baumr</author>
 <class>TranslatorConfigUi</class>
 <widget class="QWidget" name="TranslatorConfigUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>893</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="language">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_secondary">
        <property name="text">
         <string>Default source language</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="primaryLanguage">
        <property name="toolTip">
         <string>Will be used if no source language is specified.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_primary">
        <property name="text">
         <string>Alternative source language</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="secondaryLanguage">
        <property name="toolTip">
         <string>Will be used if target language and default source language are identical.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="KMessageWidget" name="noEngineWarning">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;All translation engines are disabled. Enable at least one translation engine to see results.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="closeButtonVisible">
      <bool>false</bool>
     </property>
     <property name="messageType">
      <enum>KMessageWidget::Error</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KMessageWidget" name="bingWarningOnlyEngine">
     <property name="text">
      <string>It's not recommended to activate Bing as the only translation engine.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="closeButtonVisible">
      <bool>false</bool>
     </property>
     <property name="messageType">
      <enum>KMessageWidget::Warning</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="googleTab">
      <attribute name="title">
       <string>Google Translate</string>
      </attribute>
      <widget class="QGroupBox" name="google">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>106</width>
         <height>48</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QCheckBox" name="googleEnable">
          <property name="text">
           <string>Enable</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="bingTab">
      <attribute name="title">
       <string>Bing Translator</string>
      </attribute>
      <widget class="QGroupBox" name="bing">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>481</width>
         <height>121</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="rowWrapPolicy">
         <enum>QFormLayout::WrapLongRows</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0" colspan="2">
         <widget class="KMessageWidget" name="bingWarningReliability">
          <property name="text">
           <string>Bing Translate works unreliably. It will only sometimes show results.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="closeButtonVisible">
           <bool>false</bool>
          </property>
          <property name="messageType">
           <enum>KMessageWidget::Warning</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="bingEnable">
          <property name="text">
           <string>Enable</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="deeplTab">
      <attribute name="title">
       <string>Deepl</string>
      </attribute>
      <widget class="QGroupBox" name="deepl">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>500</width>
         <height>172</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QFormLayout" name="deepl_title_form_layout">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item row="0" column="0" colspan="2">
         <layout class="QFormLayout" name="deepl_config_form_layout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item row="0" column="1">
           <widget class="QCheckBox" name="deeplEnable">
            <property name="text">
             <string>Enable</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="deepl_api_key_label">
            <property name="text">
             <string>API Key</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="deeplApiKey"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="baiduTab">
      <attribute name="title">
       <string>Baidu</string>
      </attribute>
      <widget class="QGroupBox" name="baidu">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>500</width>
         <height>172</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QFormLayout" name="baidu_5">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item row="0" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_10">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>App ID</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="baiduAPPID"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>API Key</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="baiduApiKey"/>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="baiduEnable">
            <property name="text">
             <string>Enable</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Youdao</string>
      </attribute>
      <widget class="QGroupBox" name="youdao">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>500</width>
         <height>290</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QFormLayout" name="baidu_7">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item row="0" column="0" colspan="2">
         <layout class="QFormLayout" name="formLayout_12">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item row="0" column="1">
           <widget class="QCheckBox" name="youdaoEnable">
            <property name="text">
             <string>Enable</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>App ID</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="youdaoAPPID"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>App Secret</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="youdaoAppSec"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
